charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -190,6 +203,7 @@ spec:
         - --etcd-defrag-timeout={{ .Values.etcd.etcdDefragTimeout }}
 {{- end }}
         - --snapstore-temp-directory={{ .Values.backup.snapstoreTempDir }}
+        - --etcd-pid-command=ps ax -o pid,comm | grep 'etcd$' | awk '{print $1}' | { grep -Eo '[0-9]{1,}' || true; }
```

Comment:
See https://github.com/gardener/etcd-backup-restore/pull/383#discussion_r728852543

Reply:
As agreed in the other PR, will try to use `go-ps` as suggested. There should still be a parameter I believe, for the etcd process name (defaulting to `etcd`).

Reply:
Thanks a lot @stoyanr